integrations/Vero/browser.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import { isDefinedAndNotNull } from "../utils/commonUtils";
+import ScriptLoader from "../ScriptLoader";
+
+class Vero {
+  constructor(config) {
+    this.apiKey = config.apiKey;
+    this.name = NAME;
+  }
+
+  init() {
+    logger.debug("===In init Vero===");
+    window._veroq = window._veroq || [];
+    ScriptLoader(null, "https://d3qxef4rp70elm.cloudfront.net/m.js", true);
+    window._veroq.push(["init", { api_key: this.apiKey }]);
+  }
+
+  isLoaded() {
+    logger.debug("in Vero isLoaded");
+    return !!window._veroq;
+  }
+
+  isReady() {
+    logger.debug("in Vero isReady");
+    return !!window._veroq && !!window._veroq.ready;
+  }
+
+  /**
+   * AddOrRemoveTags.
+   * This block handles any tag addition or removal requests.
+   * Ref - http://developers.getvero.com/?javascript#tags
+   *
+   * @param {Object} tags
+   */
+  addOrRemoveTags(tags, userId) {
+    const addTags = get(tags, "add");
+    const removeTags = get(tags, "remove");
```

Comment:
Don't we have to validate them similarly to the cloud mode integration? CC @utsabc

Reply:
The examples also show sending empty arrays for sections that do not have any data.
![image](https://user-images.githubusercontent.com/88789928/177265143-083f63bf-53a7-42ee-ba3e-9140a8cbe632.png)

Reply:
Yes we should atleast the type should be array if at all empty

Reply:
We can default them to `[]` if that's what Vero expects. Just wanted to make sure we are aware of them.

Reply:
Done